test/python/v1/rebuild/features/rebuild_history.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Rebuild history
+    Background:
+        Given the io-engine is running and a volume is created with more than one replicas
```

Comment:
why are there 2 different files that seem the same?

Reply:
Presently I see under tests/ directory two paths where files are duplicated. One directory path is under v1, similar to how grpc code is laid out. I followed that.

Reply:
We don't need to update "v0" anymore. Only need to add new features for v1.

Reply:
Updated accordingly, please take a look. The py file is marked as "to be implemented" which will be completed with GTM-352.